### ISSUE TITLE
license:aggregate-add-third-party -- is aggregateMissingLicensesFile used?

### ISSUE THREAD

**Issue by dhalperi** [mojohaus/license-maven-plugin#95]
The default value (of `${project.basedir}/THIRD-PARTY.properties`) seems good, and I've created that file with the right content. However, it does not seem to be used.

```
[WARNING] There is 2 dependencies with no license :
[WARNING]  - com.microsoft.z3--z3--batfish-internal
[WARNING]  - org.glassfish.grizzly--grizzly-npn-api--1.1
```

* when I don't set `-Dlicense.useMissingFile`, I get warnings about unknown licenses.
* when I do set it, I get the same warnings, but it also still creates per-module `src/license/THIRD-PARTY.properties` files.

Here's my file:

```
% cat THIRD-PARTY.properties 
com.microsoft.z3--z3--batfish-internal=MIT License
org.glassfish.grizzly--grizzly-npn-api--1.1=CDDL+GPL
```

**Comment by dhalperi** [comment]
The property seems to be defined by never used in this repository: https://github.com/mojohaus/license-maven-plugin/search?utf8=%E2%9C%93&q=aggregateMissingLicensesFile&type=

**Comment by tchemit** [comment]
@dhalperi , this parameter was just renamed to **missingLicensesFile** (see #93).
I close this issue since it will be resolved by #93.
